website/server/controllers/api-v3/user.js

```diff
@@ -1326,4 +1326,33 @@ api.setCustomDayStart = {
   },
 };
 
+/**
+* @api {post} /user/notifications/clear/:invitedUsername/:groupId Clear a notification of an accepted invite
```

Comment:
Sorry, when these comments were made I was waiting for Blade to look through it as well since it looked like all the comments made above were minor spelling or wrong comment fixes but I should've said something here. I'm working on this now and will resubmit again once I've fixed this.